docs/update_wiki_automatically.md

```diff
@@ -0,0 +1,72 @@
+# Updating Wiki with docs files automatically on every push
+
+1. create workflow and specify when it should dispatch
+2. create a job
+3. list the steps
+
+## creating workflow
+Firstly, you should name the workflow, and specify with `on` when it is dispatched. In our case, whenever you push something on the main branch the action will activate.
+```
+name: Update Wiki
+
+on:
+  workflow_dispatch:
+  push:
+    branches:
+      - main
+```
+## create a job
+Normally when you create a workflow there are list of jobs needed, but in our case, as the code is simple, only one was enough. Also `deploy` is necessary to deploy your code to a production or staging environment. We need to add operating system, for that we use `runs-on`.
+```
+jobs:
+  deploy:
+    runs-on: ubuntu-latest
+``` 
+
+## steps
+We need to clone 2 repos, copy docs folder from our repo to wiki, commit and push changes. Now discussing each step in the list below.
+
+1. first step is to clone the repo we work in and because of that we have all the information needed so we don't use `with`.
+```
+      - name: Clone Repo
+        uses: actions/checkout@v3
+```
+2. now we need to clone wiki repo. Extra information is needed, but as we only need to add folder, repository and ssh-key will be enough. also `path` is used to clone repo in that folder.
```

Comment:
I believe we need to create a clone of our repo IN the wiki, not the other way

Reply:
why is that?